packages/config/src/bin/serve-chrome.ts

```diff
@@ -0,0 +1,164 @@
+import axios from 'axios';
+import path from 'path';
+import fs from 'fs';
+import { execSync, spawn } from 'child_process';
+import { load } from 'js-yaml';
+import { LogType, fecLogger } from '@redhat-cloud-services/frontend-components-config-utilities';
+import waitOn from 'wait-on';
+
+const REPO_OWNER = 'RedHatInsights';
+const REPO_NAME = 'insights-chrome';
+const CONTAINER_PORT = 8000;
+export const EXPOSED_PORT = 9999;
+
+type ContainerRuntime = 'docker' | 'podman';
+let execBin: ContainerRuntime | undefined = undefined;
+
+const chromeDeploymentConfig = {
+  repo: 'git@gitlab.cee.redhat.com:service/app-interface.git',
+  deployFile: 'data/services/insights/frontend-base/deploy.yml',
+  stablePath: 'resourceTemplates[0].targets[4].ref',
```

Comment:
Getting a specific target by an index is very rigid.

Reply:
This is not used I'll remove it.